### PR TITLE
Changes made to the toggle button

### PR DESCRIPTION
**Pull request by Priyansh61**
The toggle button has been changed to match the background

### New toggle button :
image

### CODE REVIEW

**Comment by immattdavison** [comment]
Questioning why it matches the background? The button needs to contrast in white or the orange red to contrast the background and show clearly.

File: style.css [hunk 1]
```diff
@@ -49,7 +49,8 @@ body {
   left: 0;
   right: 0;
   bottom: 0;
-  background-color: red;
+  min-width:80px;
+  background-color: #007bff;
```
Comment (immattdavison) [inline style.css:53]: Matching the body background means the button is invisible.

**Review by immattdavison — CHANGES_REQUESTED** [review]

File: style.css (hunk 1 above)
Comment (Priyansh61) [reply style.css]: @immattdavison  I have changed the button background to orange and slider to blue, to make it easily distinguishable.

**Comment by Priyansh61** [comment]
Hey @immattdavison  PTAL! at the new changes

**Comment by immattdavison** [comment]
> Hey @immattdavison PTAL! at the new changes

You will need to resolve merge conflicts first, please.

**Comment by Priyansh61** [comment]
I am closing this PR since these issues have been resolved in the new PR.

Sorry for the inconvenience.

**Pull request closed without merging**